scripts/process-export.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+"""
+Script for processing an exported CSV from the "Combined" tab of the current
```

Comment:
It would be good to expand on exactly what's being done, rather than just "processing"

Reply:
4059602